### PR TITLE
Update GA4 analytics

### PR DESCRIPTION
**Pull request by corvidism**
Change the keys in the Google Analytics 4 page view hit to correct names with a `page_*` prefix and add new keys `page_route`, `page_status`, `page_referrer` - last of which is manually tracked to account for IMA's SPA navigation.

### CODE REVIEW

File: packages/plugin-analytic-google/src/GoogleAnalytics4.ts [hunk 1]
```diff
@@ -47,6 +48,8 @@ export class GoogleAnalytics4 extends AbstractAnalytic {
 
   /**
    * Initializes the Google Analytics 4 plugin.
+   * @param config
```
Comment (Rayus7) [inline packages/plugin-analytic-google/src/GoogleAnalytics4.ts:51]: we can remove these params from jsdoc, there is no added value..

**Review by Rayus7 — APPROVED** [review]

File: packages/plugin-analytic-google/src/GoogleAnalytics4.js [hunk 2]
```diff
@@ -137,16 +137,16 @@ export default class GoogleAnalytics4 extends AbstractAnalytic {
    */
   _getPageViewData(pageData) {
     const page_location = this._window.getUrl();
```
Comment (jenahajek) [inline packages/plugin-analytic-google/src/GoogleAnalytics4.js]: Nechceme ty optional chaining operatory i tady a dole v te metode `_createGlobalDefinition()`?

Comment (corvidism) [reply packages/plugin-analytic-google/src/GoogleAnalytics4.js]: Tady jsem dala, ale v `_createGlobalDefinition()` to pak vůbec nedává smysl. Mohla bych tam dát prázdný return, ale zdá se mi lepší, když to přímo spadne.

**Review by jenahajek — APPROVED** [review]

File: packages/plugin-analytic-google/src/GoogleAnalytics4.ts (hunk 1 above)
Comment (corvidism) [reply packages/plugin-analytic-google/src/GoogleAnalytics4.ts]: (After discussion I disabled the rules and removed the defs.)

**Pull request merged**